extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/CustomAction.scala

```diff
@@ -34,26 +34,31 @@ case class CustomActionRequest(name: String,
 
 case class CustomActionResult(req: CustomActionRequest, msg: String)
 
-sealed trait CustomActionError extends Exception {
+abstract class CustomActionError extends Exception with Error {
```

Comment:
Why we need both `CustomActionError` and `Error`. What benefits do you see in this approach?